pages/[id].js

```diff
@@ -0,0 +1,59 @@
+import KidProfile from "@/components/KidProfile";
+import useSWR from "swr";
+import { useRouter } from "next/router";
+
+export default function KidDetails() {
+  const router = useRouter();
+  const { id } = router.query;
+  const { data: kidData, isLoading, mutate} = useSWR(`/api/kids/${id}`);
+  console.log(kidData);
```

Comment:
Whilst i am very happy to see console.logs in the code, we probably don't want them making it into production. 
```suggestion
```